quran/quran.com-frontend-next
# fix QCF font

### Summary
- Fixed QCF font rendering issue. Last value of lodash [range](https://lodash.com/docs/4.17.15#range) wasn't inclusive. [1,1]  or [114,114] would yield an empty array. And no font face would generate. 
- Justify center ayah for few pages to match the printed Mushaf layout.

### Test Plan

1. Go to `{url}/114`.
2. Change the layout to page view
3. Last two lines should be centered align

### Screenshots

| Before | After |
| ------ | ------ |
| ![Before](https://user-images.githubusercontent.com/701567/126873494-54eb76b7-94f2-497b-b122-6431e3506484.png) |![After](https://user-images.githubusercontent.com/701567/126873503-f57b50e1-6291-4339-b7db-9acba49e8307.png)|
